### PR TITLE
Home work lesson5

### PR DESCRIPTION
**Pull request by efilipchik**
Check my homework please

### CODE REVIEW

File: src/Lesson6/Task21.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        for (int i = 0; i < 1000; i++) {
+            str1 = str1 + i;
+        }
+        System.out.println("Скорость выполнения String " + (System.currentTimeMillis()-startTime) + " миллисекунд");
+
+        for (int i = 0; i < 1000; i++) {
+            str2.append(i);
+        }
+        System.out.println("Скорость выполнения StringBuilder " + (System.currentTimeMillis()-startTime) + " миллисекунд");
```
Comment (RomanKashenok) [inline src/Lesson6/Task21.java]: startTime should be assigned before every cycle.

File: src/Lesson6_add/Task2_add.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package Lesson6_add;
+
+public class Task2_add {
+    /**Заменить все вхождения символа стоящего в позиции 3 на символ стоящий в позиции 0
+     */
+    public static void main(String[] args) {
+        String str = new String("Java Java Java");
+        char a = str.charAt(0);
+        char b = str.charAt(3);
+        String str2 = str.replace(b, a);
```
Comment (RomanKashenok) [inline src/Lesson6_add/Task2_add.java:10]: wrong method. This way will be replaced only first character

**Review by RomanKashenok — CHANGES_REQUESTED** [review]
minor fixes required

File: src/Lesson6_add/Task2_add.java (hunk 2 above)
Comment (efilipchik) [reply src/Lesson6_add/Task2_add.java:10]: I don't understand.
Source string: Java Java Java
Final string: JJvJ JJvJ JJvJ
All characters replaced

Comment (RomanKashenok) [reply src/Lesson6_add/Task2_add.java:10]: Sorry ) all is fine )

**Pull request merged**